README.md

```diff
@@ -26,13 +26,21 @@ The schema definitions are in [internal/schemas.ts](internal/schemas.ts).
 
 After editing the schemas, re-generate the language-specific definitions by running `yarn update-generated-files`.
 
-### Releasing to NPM
+### Release instructions
 
-1. Update the version number in package.json, make a PR, and merge to main
+#### TypeScript
+
+1. Update the version number in `package.json`, make a PR, and merge to main
 2. Make a git tag of the form `releases/typescript/vX.Y.Z` on the squash-merged commit, and push the tag
 3. GitHub Actions will take care of the rest
 
-### Releasing foxglove_msgs for ROS
+#### Python
+
+1. Update the version number in `python/[package-name]/setup.cfg`, make a PR, and merge to main
```

Comment:
the version number should not be updated in `setup.cfg`:
```suggestion
1. Update the `__version__` field in in `python/[package-name]/<main module name>/__init__.py`, make a PR, and merge to main.
```

Currently `setup.cfg` references this attribute.

Reply:
That doesn’t seem to be true: https://github.com/foxglove/schemas/blob/main/python/foxglove-schemas-protobuf/setup.cfg

Reply:
oh i see, this isn't in `foxglove/mcap`.